.scalafmt.conf

```diff
@@ -0,0 +1,6 @@
+runner.dialect = scala3
+version=3.0.1
+maxColumn=180
+project.git = true
+align.preset = more
```

Comment:
Let's go back to `some` for this one. I'm not comfortable with all the alignments of assignments that are happening.

Reply:
Done rebased